### ISSUE TITLE
New options

### ISSUE THREAD

**Issue by maybelinot** [kejbaly2/rafflepy#2]
@kejbaly2 what do you think about adding of these options?
- Make set() from input list() of usernames to get rid of duplicates ( equalizes chances of winning)
- Allow / disallow repeats of the same username (infinite ticket)

**Comment by maybelinot** [comment]
@kejbaly2 , now you have 4 options
e - equalize
i - infinite
- without options
  - Select from list of usernames, each username can win only once (duplication allowed)
- -i
  - Select from list of usernames, each username can win any number of attempts(more duplicates of usernames - more chance of winning)
- -e
  - Select from set of usernames, each username can win only once (every person can win only once)
- -ie
  - Select from set of usernames, each username can win any number of attempts(count of duplicates has no influence on chance of winning)
